modules/administration/pages/content-lifecycle.adoc

```diff
@@ -30,15 +30,56 @@ The software channels you selected should now be showing.
 . Click btn:[Attach/Detach Filters].
 . In the [guimenu]``Filters`` dialog, select the filters you want to attach to the project.
 To create a new filter, click btn:[Create new Filter].
-. Click btn:[Add new Environment].
+. Click btn:[Add Environment].
 . In the [guimenu]``Environment Lifecycle`` dialog, give the first environment a name and a description, and click btn:[Save].
-The [guimenu]``name`` field only accepts lowercase letters, numbers, periods (``.``), hyphens (``-``) and underscores (``_``).
+The [guimenu]``Name`` field only accepts lowercase letters, numbers, periods (``.``), hyphens (``-``) and underscores (``_``).
 . Continue creating environments until you have all the  environments for your lifecycle completed.
 You can select the order of the environments in the lifecycle by selecting an environment in the [guimenu]``Insert before`` field when you create it.
 
-////
+[NOTE]
+.Content Filters
+====
+Content Filters are global in your organization and can be shared in multiple projects.
+====
+
+[NOTE]
+.Adding environment in existing project
+====
+When you add an environment to an existing project that has already some sources built, the newly added environment will be populated with the content of its predecessor environment.
+====
+
 .Procedure: Using a Content Lifecycle Project
 
-. TBD ...
 
-////
+=== Building Content Project
+Once you create a project, define environments, attach sources (and possibly filters), you can built the project for the first time.
+
+Building applies filters to the attached sources and clones them to the first environment in the project.
+
+. In the {productname} {webui}, navigate to menu:Content Lifecycle[Content Lifecycle Projects], and select the project you wish to build
+. Review the attached sources and filters
+. Click btn:[Build]
+. Check the build progress in the [guimenu]``Environment Lifecycle`` section
+
+After the build is finished, the environment version is increased by one and the built sources (e.g. software channels) can be assigned to systems.
```

Comment:
:arrow_right `After the build is finished, the first environment version is increased by one as it will contain the latest built sources ready to be used afterwards (e.g. new software channels can be assigned to systems).`

Reply:
Hmm, I find the level of confusion of the newer version slighly higher then in the original :/